Lukem/index page node (#214)
moondance-labs/tanssi-docs: node-operators/network-node/index.md

---

> title: Network Node

Reviewer: ```suggestion
title: Run a Node on a Tanssi-Powered Network
```

This title is for SEO purposes, so let's try to add a little bit more context in these titles when possible. But one thing to keep in mind is that these titles can be no more than 60 characters, and ` | Tanssi Docs` is appended to the end of the titles automatically. So you really have like 46 characters to work with

> description: Learn how to set up and run a Tanssi-powered network using Docker or Systemd, which allows you to have your own RPC endpoint to interact with your appchain.

Reviewer: ```suggestion
description: Learn how to set up and run a node on a Tanssi-powered network using Docker or Systemd, which allows you to host your own RPC endpoint for chain interaction.
```

We have to avoid using appchain now
